obspy/io/ah/core.py

```diff
@@ -228,6 +227,217 @@ def _unpack_trace(data):
         return st
 
 
+def _write_ah1(stream, filename):
+    """
+    Writes a Stream object to an AH v1 waveform file.
+
+    :type stream:
+    :param stream: The ObsPy Stream object to write.
+    :type filename: str
+    :param filename: open file, or file-like object
+
+    """
+    def _pack_trace_with_ah_dict(trace, packer):
```

Comment:
Could you move both out of the `_write_ah1()` function? This is IMHO better style and would potentially allow their direct usage/unit testing.